### PR TITLE
"Ik heb de database met het users tabel gemaakt, en deze heb ik nog n…

### PR DESCRIPTION
**Pull request by BenBerkenbosch**
…iet gevuld met mockdata. Er zit een isAdmin boolean in."

### CODE REVIEW

**Comment by keygun-development** [comment]
Nice commit message btw

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -17,17 +17,16 @@ public function up()
...
+            $table->boolean('isAdmin');
             $table->timestamps();
         });
     }
 
     /**
      * Reverse the migrations.
      *
-     * @return void
+     * @return voidk
```
Comment (keygun-development) [inline database/migrations/2014_10_12_000000_create_users_table.php:29]: Revert this line please

**Review by keygun-development — CHANGES_REQUESTED** [review]

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 1 above)
Comment (BenBerkenbosch) [reply database/migrations/2014_10_12_000000_create_users_table.php:29]: Yes sir, with pleasure!

**Comment by keygun-development** [comment]
Next time try to shorten your commit message and only provide the things you've done. Thanks in advance! Will see you later on school today while we have "mentorles"

**Review by keygun-development — APPROVED** [review]

**Comment by BenBerkenbosch** [comment]
My sincere apologies, Keygun, next time I will keep this in mind before comitting, as this may disrupt your usual workflow, which is never my intention. Your feedback is most appreciated, and I hope you have a wonderful day!

**Comment by BenBerkenbosch** [comment]
If you don't mind me asking, I would appreciate it if you could provide me with a new task, I may not be particularily helpful with everything, but apart from my extensive use of the English language, I am also highly capable of trying something new and put my best effort in getting the expected results.

**Pull request merged**